Dockerfile.node7

```diff
@@ -0,0 +1,14 @@
+# Dockerfile for using Yarn along with Node.js 7.x
+# docker build -t yarnpkg/node-yarn:latest -f Dockerfile.node7 .
```

Comment:
No, you'll still have all the layers locally, it's just the resulting image that's a single layer :smile: